src/Geom_Poly.f90

```diff
@@ -156,6 +158,13 @@ MODULE Geom_Poly
   !> @copydetails Geom_Poly::isequal_PolygonType
   MODULE PROCEDURE isequal_PolygonType
 ENDINTERFACE
+
+!> @brief
+INTERFACE ASSIGNMENT(=)
+  !> @copybrief Geom_Poly::assign_PolygonType
+  !> @copydetails Geom_Poly::assign_PolygonType
+  MODULE PROCEDURE assign_PolygonType
+ENDINTERFACE
```

Comment:
Same question here about the necessity of assignment

Reply:
The other two are probably fine to delete.  The polygontype does have pointer assignments to other polygontypes, so this one might be good to keep.

Reply:
Ah, good point.   I missed those 2 pointers at the bottom.  So yeah, definitely keep this one.